### ISSUE TITLE
Any plan to support SSR ? 

### ISSUE THREAD

**Issue by nzhl** [nhn/tui.editor#875]
It seems current editor is not working in SSR situation ( used window somewhere ).

**Comment by seonim-ryu** [comment]
@N6REJ We have no plans to apply for SSR yet. And "used window somewhere" what is this talking about? I'm curious.

**Comment by nzhl** [comment]
> @N6REJ We have no plans to apply for SSR yet. And "used window somewhere" what is this talking about? I'm curious.

When I tried to use sever side rendering, a error info indicates that the global object window (should only exist in browser env) is not defined. So i guess it has been used in somewhere. Sorry to be obscure ~

**Comment by hrzon** [comment]
@nzhl I've got the same error with Next.js. I don't know whether you use Next.js or not, but if you prevent tui.editor from being loaded on node you don't see the error (of coursely tui.editor will not be rendered on server side, but I think there is little advantage to do it...).

For whom trying to use tui.editor with Next.js:

```tsx
import dynamic from 'next/dynamic';
import * as React from 'react';
import { EditorProps } from '@toast-ui/react-editor';

const Editor = dynamic<EditorProps>(() => import('@toast-ui/react-editor').then(m => m.Editor), { ssr: false });

const WysiwygEditor: React.FC = () => {
  return <Editor
    initialValue="hello react editor world!"
    previewStyle="vertical"
    height="600px"
    initialEditType="markdown"
    useCommandShortcut={true}
  />;
}

export default WysiwygEditor;
```

**Comment by ats1999** [comment]
> @nzhl I've got the same error with Next.js. I don't know whether you use Next.js or not, but if you prevent tui.editor from being loaded on node you don't see the error (of coursely tui.editor will not be rendered on server side, but I think there is little advantage to do it...).
> 
> For whom trying to use tui.editor with nextjs: [myeongjae-kim/tui.editor-with-nextjs@d11536e](https://github.com/myeongjae-kim/tui.editor-with-nextjs/commit/d11536e197d52881765f583754228c1d2f815cdc)

Yeah, we can use dynamic from next js to prevent server side rendering. 
Why we use server side rendering
1: fast page load
2: seo purpose

If we prevent server side rendering, then how can we achive seo
